package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^5.17.0",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "prop-types": "^15.8.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-scripts": "5.0.1",
```

Comment:
![image](https://github.com/tajemouti/math-magicians/assets/93039359/d3dd00b6-f7a9-49ff-8a83-b87aaa48cdad)

**[OPTIONAL]**

- [ ] It would be nice if you remove the Live Demo section from your readme file since you don't have any working live link in your readme file 😨 Kindly remove this whole Live Demo section to make your readme descriptive and professional 👍